File: letter_count.c
#include <stdio.h>
#include <stdlib.h>
#include <err.h>

unsigned long letter_count(FILE* fd)
{
	char c;
	unsigned long cpt = 0;

	if (!fd)
		err(1, "Unable to read directory");
	while ((c = fgetc(fd)) != EOF)
	{
		if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z'))
			cpt += 1;
	}
	return cpt;
}

int main(int argc, char** argv)
{
	if (argc != 2)
		err(1, "Invalid Argument");
	FILE* fd = fopen(argv[1], "r");
	printf("This file contains %lu letter(s)\n", letter_count(fd));
	fclose(fd);
	return 0;
}

File: letter_count.py
import sys

def letter_count(argv):
    f = open(argv, "r")
    cpt = 0
    line = f.readline()
    while line:
        if line == "":
            break
        for x in line:
            if (x >= 'a' and x <= 'z') or (x >= 'A' and x <= 'Z'):
                cpt += 1
        line = f.readline()
    f.close()
    return cpt

def main(argv):
    print("This file contains " + str(letter_count(argv)) + " letters.");


if __name__ == "__main__":
    main(sys.argv[1])
